ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/view/repository/ViewRepository.java

```diff
@@ -0,0 +1,11 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.domain.view.repository;
+
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.viewcreation.valueobjects.ViewSpecification;
+
+import java.util.List;
+
+public interface ViewRepository {
+	List<ViewSpecification> retrieveAllViews();
+
```

Comment:
maybe already provide a delete method with a notImplementedException behind it to be in line with the Service?